src/style.css

```diff
@@ -1 +1,35 @@
 /* add styles here */
+:root{
+  --main-images: (./images/star-active.svg)
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-position: center;
+  margin-right: 4px;
+  background-repeat: no-repeat;
```

Comment:
The `background-image` URL should be extracted into a custom property since it's reused in multiple places. This will make it easier to manage and update the image path if needed.